assets/tchatcha/tchatcha.php

```diff
@@ -119,6 +119,8 @@ class tx_tchatchachaform_tchatcha
 		)
 	);
 
+	public $errors;
```

Comment:
Used line 464, so let's make it public to be sure no "undefined property" can be thrown.